lisp/init-terraform.el

```diff
@@ -0,0 +1,9 @@
+;;; Terraform
+
+(maybe-require-package 'terraform-mode)
```

Comment:
The terms after this should be in a "when" block because `maybe-require-package` may return `nil`, indicating that the package is not installable in the current Emacs.